adm_program/modules/lists/lists_show.php

```diff
@@ -731,7 +736,26 @@
                     $content = $arrListValues[$member[$sqlColumnNumber]];
                 }
             }
-
+            elseif ($column->getValue('lsc_special_field') === 'mem_approved')
+            {
+                // Assign Integer to Language strings
+                if ((int) $content === 1)
```

Comment:
Is ok for me. Maybe we could think about an option for both behaviours in the module preferences. So everyone can configure how it suits in the customs assembly. Let´s discuss
For me it would not be a benefit and I would disable and delete the entries.

Ok in summary:  We optimize the dafault list handling parameter in this pull request and after merging we extend the "No" functionality as new Issue?.
Would make sense for better handling and the history of the developments. What do you think?